src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+/**
+ * An Event Task that contains a description, a from indicator and a to indicator.
+ */
+public class Event extends Task {
+
+    protected String from;
+    protected String to;
+
+    /**
+     * An Event task constructor.
+     *
+     * @param description The description of the Event task.
+     * @param from the indicated starting time.
+     * @param to the indicated ending time.
+     */
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+
+    /**
+     * Returns a String representation of the Event task, including the symbol, status icon, description,
+     * and from and to indicators.
+     *
+     * @return a String to be printed to the user.
+     */
+    @Override
+    public String toString() {
+        return ("[E]" + super.getStatusIcon() + description +
+                " (from: " + from + "to: " + to + ")");
```

Comment:
With regards to coding standard, would it be better if you start the new line using the + operator?